### DISCUSSION TITLE
strange bridging

### DISCUSSION THREAD

**Discussion by igl-x2** [OrcaSlicer/OrcaSlicer#5592] [Q&A] [answered]
Hi,

I want to print a lid for a box without any supports. Rotation is not an option so it has to have some rather large bridging.
However Orca somehow ruins the geometry on the layer just below the bridge.
Cura does slice it fine.

So how are these dark blue lines caused? I have played with nearly every available option that might be remotely connected to this.

![orca45](https://github.com/SoftFever/OrcaSlicer/assets/129790664/d3975c8c-0500-4d81-9cf7-569e2c1a4611)
![orca46](https://github.com/SoftFever/OrcaSlicer/assets/129790664/4fbc2836-4f01-49ec-a61a-a3088e628833)
![cura45](https://github.com/SoftFever/OrcaSlicer/assets/129790664/fd418b60-bf34-4d7b-a374-0d3ae1385f47)
![cura46](https://github.com/SoftFever/OrcaSlicer/assets/129790664/e671cdbf-73ef-4bfe-8751-95f632a09b70)

**Comment by vgdh** [comment]
looks like a bug to me.

**Comment by igl-x2** [comment]
Latest Orca Beta creates the same result

**Comment by igl-x2** [comment, marked as answer]
Solution:

this has happened because the geometry of the model was designed for 0.2 layer heights.
I had chosen a 0.3 first layer height, which caused this strange kind of slicing.
Setting the first layer height also to 0.2 fixed this behaviour immediately.

Not a bug of Orca!
